crates/services/upgradable-executor/src/executor.rs

```diff
@@ -234,8 +234,11 @@ impl<S, R> Executor<S, R> {
         // This update has been performed on the branch release/v0.41.9 which
         // is on top of the branch release/v0.41.8 not on master.
         ("0-41-9", 24),
+        // This update is also shadowed by the branch release/v0.41.10 which
+        // is on top of the branch release/v0.41.9 not on master.
         ("0-42-0", 25),
```

Comment:
@xgreenx not sure what to do here. do we mark this as 0.41.10 or 0.42.0?

Reply:
I think we can say that `0.41.10` is a `25` since the mainnet has been upgraded to it.

`0.42.0` is only used in devnet, so it is fine to shadow it

Reply:
what would we do in the case 0.42.0 had gone to mainnet/